src/chains/tests/test_views.py

```diff
@@ -70,6 +70,10 @@ def test_json_payload_format(self) -> None:
                         "nativeCoin": chain.prices_provider_native_coin,
                         "chainName": chain.prices_provider_chain_name,
                     },
+                    "counterfactualBalancesProvider": {
```

Comment:
Perhaps we should consider that the balances provider may be used for more than just counterfactual Safe in the future. It is otherwise very feature-specific.

What do you think about having something like:

```
balancesProvider: {
  chainName: string
  counterfactual: boolean
}
```

Reply:
I think it makes more sense to do so after looking at https://github.com/safe-global/safe-client-gateway/pull/1648.

Reply:
Good point, it makes sense to me! I was trying to distinguish it from the other provider (Coingecko) but I agree it should be enough to have a `balances` prefix, as the other one is just for prices.

Will change it, thanks!

Reply:
Done in [3e75238](https://github.com/safe-global/safe-config-service/pull/1153/commits/3e7523879c89a2481100001ee1a4378e9acfbed6)